examples/with-apollo-graphql/src/client.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ApolloProvider } from 'react-apollo';
+import { ApolloClient } from 'apollo-client';
+import { createHttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+import App from './App';
+
+// Create an ApolloClient instance
+// Pass your GraphQL endpoint to uri
+const client = new ApolloClient({
+  link: createHttpLink({
+    uri: 'https://gql-placeholder.herokuapp.com/graphql',
+  }),
+  cache: new InMemoryCache(),
```

Comment:
Working from the Apollo docs (https://www.apollographql.com/docs/react/features/server-side-rendering#store-rehydration), you'll want to swap this out for
```
cache: new InMemoryCache().restore(window.__APOLLO_STATE__)
```
might also want to add 
```
ssrForceFetchDelay: 100,
defaultOptions: {
  query: {
    fetchPolicy: 'cache-and-network',
  },
},
``` 
to avoid fetching data on the client-side that the server has already fetched.